src/components/buttons/flat-button/flat-button.js

```diff
@@ -84,29 +93,31 @@ export const FlatButton = props => {
               : getTextColor(props.tone, false, overwrittenVars)};
           }
 
-          &:hover,
-          &:focus {
-            p {
-              color: ${props.isDisabled
-                ? overwrittenVars.colorNeutral
-                : getTextColor(props.tone, true, overwrittenVars)};
-            }
-
-            svg * {
-              fill: ${props.isDisabled
-                ? overwrittenVars.colorNeutral
-                : getTextColor(props.tone, true, overwrittenVars)};
-            }
+          span + span {
+            margin-left: ${vars.spacingXs};
```

Comment:
Almost the same as using an `Inline` spacings component, but we can't use flexbox here.